lib/schedule.js

```
@@ -0,0 +1,224 @@
+                dct:creator ${sparqlEscapeUri(scheduledJob.uri)};
+                dct:created ${sparqlEscapeDateTime(now)};
+                dct:modified ${sparqlEscapeDateTime(now)};
+                task:operation ${sparqlEscapeUri(scheduledJob.operation)}.
+
+          ${sparqlEscapeUri(remoteDataObjectUri)} a nfo:RemoteDataObject;
+                mu:uuid ${sparqlEscapeString(remoteDataObjectId)};
+                nie:url ${sparqlEscapeString(scheduledJob.sourceUrl)};
+                adms:status ${sparqlEscapeUri(STATUS_READY_TO_BE_CACHED)};
+                rpioHttp:requestHeader ${sparqlEscapeUri(REQUEST_HEADER)};
+                dct:created ${sparqlEscapeDateTime(now)};
+                dct:modified ${sparqlEscapeDateTime(now)};
+                dct:creator ${sparqlEscapeUri(scheduledJob.uri)}.
+
+          ${sparqlEscapeUri(harvestingCollectionUri)} a hrvst:HarvestingCollection;
+                mu:uuid ${sparqlEscapeString(harvestingCollectionId)};
+                dct:creator ${sparqlEscapeUri(scheduledJob.uri)};
+                dct:hasPart ${sparqlEscapeUri(remoteDataObjectUri)}.
+
+          ${sparqlEscapeUri(dataContainerUri)} a nfo:DataContainer;
+                mu:uuid ${sparqlEscapeString(dataContainerId)};
+                task:hasHarvestingCollection ${sparqlEscapeUri(harvestingCollectionUri)}.
+
+          ${sparqlEscapeUri(taskUri)} a task:Task;
+                mu:uuid ${sparqlEscapeString(taskId)};
+                adms:status ${sparqlEscapeUri(STATUS_SCHEDULED)};
+                dct:created ${sparqlEscapeDateTime(now)};
+                dct:modified ${sparqlEscapeDateTime(now)};
+                task:operation ${sparqlEscapeUri(scheduledJob.taskOperation)};
+                task:index ${sparqlEscapeInt(0)};
+                task:inputContainer ${sparqlEscapeUri(dataContainerUri)};
+                dct:isPartOf ${sparqlEscapeUri(jobUri)}.
+        }
+      }
+    `;
+    return newJobQuery;
+  } catch(err) {
```

Would suggest; No silent errors, or at least not that deep.